src/style.css

```
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+.bigSearch {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+/* Styles for big input */
+body {
+  margin: 0 8px;
+}
+
+.inputBigSearch {
```

```suggestion
.input-big-search {
```
it's better to name classes in kebab-case